frontend/src/main/scala/bloop/data/Project.scala

```diff
@@ -360,6 +360,12 @@ object Project {
     val workspaceDir = project.workspaceDirectory.getOrElse(configDir.getParent)
     val isDotty = project.scalaInstance.exists(_.isDotty)
 
+    def isAfterScala3M3(version: String) = {
```

Comment:
Perhaps call this
```suggestion
    def isAtLeastScala3M3(version: String) = {
```
?

Reply:
Good idea! Fixed.